doc/effective/styling.md

```diff
@@ -0,0 +1,230 @@
+# Effective Angular: Styling
+
+AngularDart offers a variety of methods for styling your application. This
+document is intended to help developers understand how and when to use these
+options.
+
+* [Global styles](#global-styles)
+  * [DO declare application-wide styles in
+    index.html](#do-declare-application-wide-styles-in-indexhtml)
+  * [DO include external style sheets in
+    index.html](#do-include-external-style-sheets-in-indexhtml)
+* [Component styles](#component-styles)
+  * [Understanding style encapsulation](#understanding-style-encapsulation)
+  * [DO use component styles where
+    applicable](#do-use-component-styles-where-applicable)
+  * [DO reuse styles in shared style
+    sheets](#do-reuse-styles-in-shared-style-sheets)
+  * [DO use host selectors to style host
+    elements](#do-use-host-selectors-to-style-host-elements)
+  * [AVOID excessive use of ::ng-deep](#avoid-excessive-use-of-ng-deep)
+  * [AVOID using ::ng-deep as the left-most
+    selector](#avoid-using-ng-deep-as-the-left-most-selector)
+
+## Global styles
+
+You may include style sheets or declarations in *index.html* just like you would
+for any other web page. We refer to these as *global styles*, because they apply
+to all of the components in your application.
+
+### DO declare application-wide styles in index.html
+
+Declare styles applicable to your entire application using a `<style>` tag in
+*index.html*.
+
+```html
+<html>
+  <head>
+    <script defer src="main.dart.js"></script>
+    <style>
+      * {
+        box-sizing: border-box;
+      }
+      body {
+        background-color: #f2f2f2;
+        font-family: Roboto, sans-serif;
+        font-size: 16px;
+      }
+    </style>
+  </head>
+  <body></body>
+</html>
+```
+
+### DO include external style sheets in index.html
```

Comment:
Maybe combine this with the previous section, because upon reading it made it seem like application styles should all be `<style>` and external ones should all be `<link>`.

They are sort of interchangable

Reply:
That message was actually my intention.

I thought I read that declaring styles in a `<script>` tag in the HTML document was preferable to including them in a `<link>` if you could. Something about reducing the number of resources to download. Now I can't seem to find the source, so I skeptical this optimization (if it even is one) is worth mentioning.